src/components/table/LogTable.vue

```diff
@@ -59,17 +59,26 @@
 <script setup lang="ts">
 import type { ILogEntry, ISortingOptions } from '../../interfaces'
 
-import { computed, nextTick, ref } from 'vue'
+import { computed, nextTick, onMounted, onBeforeUnmount, ref } from 'vue'
 import { translate as t } from '@nextcloud/l10n'
 import { useSettingsStore } from '../../store/settings'
 import { useLogStore } from '../../store/logging'
+import { debounce } from '../../utils/debounce'
+import { logger } from '../../utils/logger'
 
 import IntersectionObserver from '../IntersectionObserver.vue'
 import LogDetailsModal from '../LogDetailsModal.vue'
 import LogTableHeader from './LogTableHeader.vue'
 import LogTableRow from './LogTableRow.vue'
 import LogSearch from '../LogSearch.vue'
 
+// Items to render before and after the visible area
+const bufferItems = 3
+// Fixed height of LogTableRow
+const itemHeight = 42
```

Comment:
Absolutely ignore this comment if it's not helpful (I'm not a Vue dev, but I've worked on similar things in Flutter), but this and `theadHeight` may not be constant if I understand the implementation. 

For example, some users may opt to increase the scale of their browser's content on high DPI displays or to aid readability. I can imagine scaling much beyond the assumed heights here would stop scrolling working as intended. 

Of course, CSS and other elements may change outside of this file too in future commits.
 
![image](https://github.com/user-attachments/assets/ae66b249-d060-4e30-9cec-a67c81743742)

Reply:
Good point! Although actual amount of screen pixels taken might vary, from CSS perspective it's still the same parameters: `font-size: 15px`, button `height: 34px`. Current logic modifies and relies on CSS values, so should be fine.

Adding a resize check in case text wrapping may happen (hope it's not an overhead), but default should be aligned in the code anyway (starting `ref(value)` and style's `min-height`)